src/MutableDateTime.php

```diff
@@ -47,6 +47,8 @@
  * @property-read bool $utc checks if the timezone is UTC, true if UTC, false otherwise
  * @property-read string $timezoneName
  * @property-read string $tzName
+ * @property-read string $date used in opis/closure serialization via reflection
+ * @property-read int $timezone_type used in opis/closure serialization via reflection
```

Comment:
Those are only for test though?

Reply:
yes, and used internally via Reflection by opis/closure

Reply:
But only in require-dev (testing). IMO that must not leak into the actual classes, only confuses to have those there and unused, right?
You could make a TestMutableDateTime that adds them on top, for example.

Reply:
https://github.com/opis/closure/blob/master/src/SerializableClosure.php#L403 there

Reply:
OK, I'll create test classes and keep Stan happy, any concerns with the actual solution before fixing Stan and php5.5 (hopefully) ?

Reply:
I am confused. How do these new properties fix serialization? Are we adding setters for properties that were readonly before?

Reply:
@markstory the serialization done by opis/closure relies in Reflection to extract the instance properties, https://gist.github.com/steinkel/80bfdb2755cbc8cc722c1ab7559f398a and Reflection is returning "date", "timezone_type", as ReflectionProperty, possibly because of > https://bugs.php.net/bug.php?id=75232. We don't want or need these new setter exposed in docs (will get this fixed as suggested by @dereuromark), but we would need them to allow opis work with this class (or any other class extending \DateTime)